fixtures/remix-pages-app/package.json

```diff
@@ -15,23 +15,23 @@
 		"test:watch": "npx vitest"
 	},
 	"dependencies": {
-		"@remix-run/cloudflare-pages": "^1.8.2",
+		"@remix-run/cloudflare-pages": "1.16.1",
```

Comment:
1.17.0?

Reply:
I was just trying to get it working with as little churn as possible; and the current highest version that was in the package.json was 1.16.2.
Feel free to bump to 1.17.0 in a follow up.

Reply:
Huh? I thought made them all 1.16.2, right? Which ones are inconsistent?

Reply:
OMG! I just saw what you mean.